src/sys/unix/mod.rs

```diff
@@ -50,7 +50,7 @@ pub use iovec::IoVec;
 
 use std::os::unix::io::FromRawFd;
 
-pub fn pipe() -> ::io::Result<(Io, Io)> {
+pub fn pipe() -> std::io::Result<(Io, Io)> {
```

Comment:
Shouldn't this be `crate::io::Result<(Io, Io)>`? It's the same re-exported type, but I mean just for consistency...

Reply:
I think the `io` mod w/ the re-exports should go away. I am just leaving that for a different PR.